openedx/features/partners/give2asia/views.py

```diff
@@ -164,6 +165,15 @@ def create_account_with_params_custom(request, params, partner):
     # Perform operations that are non-critical parts of account creation
     preferences_api.set_user_preference(user, LANGUAGE_KEY, get_language())
 
+    try:
+        partner_communities = PartnerCommunity.objects.filter(partner=partner)
+        for community in partner_communities:
+            task_join_group_on_nodebb.delay(
+                category_id=community.category_id, username=user.username)
+    except Exception as err:
+        log.exception("Community joining failed for user {username} for partner {slug}"
```

Comment:
can we add community_id/category_id to the log statement?

Reply:
Done